config/settings.yml

```diff
@@ -27,3 +27,11 @@ plugins:
     client: true
     default: 10
     min: 1
+  assign_allowed_on_groups:
```

Comment:
I'm not sure about this as a site setting and I feel like this should be a group option stored in a group custom fields. The trouble with a site setting is that when a group is renamed or deleted, the site settings becomes inaccurate. Having to handle that in a nice manner isn't going to be easy.

Reply:
I think that the site setting makes it easier to manage multiple allowed groups. Maybe we can track when a group gets renamed/deleted and make sure it remains synced?

Reply:
Hmm maybe but I still think that having to maintain this sync is quite tricky. Maybe have a look how [type: category](https://github.com/discourse/discourse/blob/e2444e0d31588e373a018b5321b40cd3c56d7267/config/site_settings.yml#L674-L676) for site settings is handled? I'm not sure myself but worth exploring.

Reply:
Ended up using `type: group_list` instead.